### PR TITLE
Added logic to support git+ssh repo urls.

### PR DESCRIPTION
**Pull request by bk-dev**
Our team needs to be able to install an SFDX plugin via a git+ssh repo url. Hopefully, this is a decent solution to support this within the oclif project.

### CODE REVIEW

**Comment by bk-dev** [comment]
@jdxcode Hello, there. Could you help me reconcile this PR? I'm not sure what I have to do at this point, but the proposed change is needed by my team.

File: src/commands/plugins/install.ts [hunk 1]
```diff
@@ -62,8 +62,10 @@ e.g. If you have a core plugin that has a 'hello' command, installing a user-ins
   }
   /* eslint-enable no-await-in-loop */
 
-  async parsePlugin(input: string): Promise<{name: string; tag: string; type: 'npm'} | {url: string; type: 'repo'}> {
-    if (input.includes('@') && input.includes('/')) {
+  async parsePlugin(input: string): Promise<{name: string, tag: string, type: 'npm'} | {url: string, type: 'repo'}> {
```
Comment (RasPhilCo) [inline src/commands/plugins/install.ts:65]:
```suggestion
  async parsePlugin(input: string): Promise<{name: string; tag: string; type: 'npm'} | {url: string; type: 'repo'}> {
```

**Comment by RasPhilCo** [comment]
@bk-dev are you able to sign the CLA?

File: src/commands/plugins/install.ts [hunk 2]
```diff
@@ -63,7 +63,9 @@ e.g. If you have a core plugin that has a 'hello' command, installing a user-ins
   /* eslint-enable no-await-in-loop */
 
   async parsePlugin(input: string): Promise<{name: string; tag: string; type: 'npm'} | {url: string; type: 'repo'}> {
-    if (input.includes('@') && input.includes('/')) {
+    if (input.startsWith('git+ssh://')) {
+      return {url: input, type: 'repo'}
+    } else if (input.includes('@') && input.includes('/')) {
```
Comment (RasPhilCo) [inline src/commands/plugins/install.ts:68]:
```suggestion
}
if (input.includes('@') && input.includes('/')) {
```

**Comment by RasPhilCo** [comment]
@bk-dev ping. If not, I'll make a PR and reference this one as the originating source.

**Pull request closed without merging**